util/wren.mk

```diff
@@ -103,9 +103,13 @@ else
 	SHARED_LIB_FLAGS := -Wl,-soname,$@.so
 	SHARED_EXT := so
 
+ifeq (,$(findstring Windows,$(OS)))
```

Comment:
I'm sure the makefile defines `OS`. I didn't realize at the time that collides with an existing Windows environment variable. The goal was never to use a value coming from the environment. The makefile wants to control the value of `OS`. We could give it another name if it's helpful.